src/Robo/Blt.php

```diff
@@ -116,6 +120,46 @@ private function addPluginsCommandsAndHooks() {
   }
 
   /**
+   * Adds Symfony (non-Robo) command classes to the application.
+   *
+   * @param \Acquia\Blt\Robo\Application $application
+   */
+  protected function addSymfonyCommands(Application $application) {
```

Comment:
I do wish we could move this out of the Blt.php file. It feels like it doesn't belong here.

Reply:
I agree, so I refactored and put everything in the TwigCommand class. I think it's cleaner now.